BouncyCastle-JCA/src/Cipher.crysl

```diff
@@ -120,7 +123,8 @@ REQUIRES
 	mode(transformation) in {"GCM"} => preparedGCM[params];
 	
 ENSURES
-	encrypted[pre_ciphertext, pre_plaintext] after updates; 
+	generatedCipher[this] after Inits;
+	encrypted[pre_ciphertext, pre_plaintext] after Updates; 
 	encrypted[cipherText, plainText];
 	encrypted[cipherBuffer, plainBuffer];
 	wrappedKey[wrappedKeyBytes, wrappedKey];
```

Comment:
If you make changes, can you add the empty line at the end of the rules? (optionaly)